### ISSUE TITLE
wk1 may_epic BUG: Team Data Table Auto Save Feature not working

### ISSUE THREAD

**Issue by lijenn** [lzim/teampsd#1903]
@anazariz This was previously a part of #1806, where the logic code was missing entirely.

The implementation did not work on the actual SharePoint site, but worked in the test folders on TeamPSD Shared Drive.
We need to test the implementation using test folders on SharePoint instead.

Refer to the MTL calendar for upcoming MTL Teams by priority.

**Comment by anazariz** [comment]
@lijenn and @dkngenda to test the solution on 4/22.

**Comment by anazariz** [comment]
@lijenn this bug fix was tested alongside the VizMeau fix/test (#1913). Can I close it?

**Comment by lijenn** [comment]
@anazariz No, it's still not working for me..let me test again later today to double check and update you.

**Comment by anazariz** [comment]
Ok, I was saying that based on Stacey's pass. Please test with one of the sites that we released this week. Thx!

**Comment by lijenn** [comment]
@anazariz It worked :)
Please go ahead and propagate the data ui file for the teams for this week and then all of the other facilities once you're able to.

**Comment by anazariz** [comment]
@lijenn, I propagated 605, 660, 657 and 589. Will do the rest as a second wave.

**The issue:** The file path in SharePoint is different that the file path in Windows. SharePoint uses UNC (Universal Naming Convention). I fixed this bug by switching from the **mapped path** (Windows) to the **UNC path**. 

However, UNC path cannot be test on a regular Windows filesystem. So I created a test folder in 640, which will be used for testing path issues going forward.  

@lijenn, can this bug ticket be closed now?
